src/grid/molgrid.py

```diff
@@ -628,3 +654,33 @@ def __getitem__(self, index: int):
                 self._atcoords[index],
             )
         return self._atgrids[index]
+
+
+def _generate_default_rgrid(atnum: int):
+    r"""
+    Generate default radial transformation grid from default Horton.
+
+    See _DEFAULT_POWER_RTRANSFORM_PARAMS inside utils for information on how
+    it was determined
+
+    Parameters
+    ----------
+    atnum: int
+        Atomic Number
+
+    Returns
+    -------
+    OneDGrid:
+        One-dimensional grid that was transformed using PowerRTransform.
+
+    """
+    if atnum in _DEFAULT_POWER_RTRANSFORM_PARAMS:
+        rmin, rmax, npt = _DEFAULT_POWER_RTRANSFORM_PARAMS[int(atnum)]
+        rmin, rmax = rmin * 1.8897259885789, rmax * 1.8897259885789
```

Comment:
1. Please use `scipy.constants.angstrom / scipy.constants.value("atomic unit of length")` for unit conversion.
2. Please add a comment that r values are converted from Angstom to Bohr.

Reply:
This is fixed in the following comments